actors/miner/src/state.rs

```diff
@@ -766,9 +766,7 @@ impl State {
 
     // Return true when the miner actor needs to continue scheduling deadline crons
     pub fn continue_deadline_cron(&self) -> bool {
-        !self.pre_commit_deposits.is_zero()
-            || !self.initial_pledge.is_zero()
-            || !self.locked_funds.is_zero()
+        !self.pre_commit_deposits.is_zero() || !self.initial_pledge.is_zero()
```

Comment:
I don't think this change is reasonable. A miner needs to keep vesting if they have locked funds - we can't tell how they reached this state.

Reply:
i have a progessing version without this code, but there are too much test case to fix.  and had time to handle this since I quit my job recently.

Reply:
hmm, check_state_invariants behavior need change.
https://github.com/filecoin-project/builtin-actors/blob/b81da94e0ff43585af3c22387a1ef6c5cca76cb2/actors/miner/src/testing.rs#L306